hal/src/electron/modem/mdm_hal.cpp

```diff
@@ -367,10 +368,10 @@ int MDMParser::sendFormattedWithArgs(const char* format, va_list args) {
     return n;
 }
 
-int MDMParser::_checkAtResponse(void)
+int MDMParser::_checkAtResponse(bool fastTimeout /* = false */)
```

Comment:
Nit: Perhaps we could simply just take the timeout value here with a default set to `AT_TIMEOUT`

Reply:
It's a good callout, and I though about it, but I would have had to move all of the defines to mdm_hal.h... and I thought it might be simpler not to :D

Reply:
A default argument can also be specified in the implementation and since these are private methods there shouldn't be any difference (there would be if it was a public method, where the default argument would only be coming into play just in the cpp file that has it and the external code wouldn't be able to use it). Fine to leave as-is :)